djangocms_bootstrap4/contrib/bootstrap4_link/constants.py

```diff
@@ -15,3 +15,9 @@
     ('', _('Medium')),
     ('btn-lg', _('Large')),
 )
+
+LINK_ICONS = getattr(
+    settings,
+    'DJANGOCMS_BOOTSTRAP4_ICONS',
```

Comment:
Maybe a better name for this var? At first sight I thought it was a list, not a boolean.

Reply:
For consistency sake I would use something like `DJANGOCMS_BOOTSTRAP4_USE_LINK_ICONS`